tests/e2e_tests/noisemap_cal_e2e.py

```diff
@@ -249,24 +417,27 @@ def test_noisemap_calibration_from_l2a(tvacdata_path, e2eoutput_path):
         # plt.savefig(os.path.join(noisemap_outputdir,f"CorgiDRP_TVAC_Comparison_l2a_to_l2a_{noise_ext}.pdf"))
         # plt.close()
 
-        assert np.all(np.abs(diff) < 1e-5)
+        # assert np.all(np.abs(diff) < 1e-5)
 
 if __name__ == "__main__":
     # Use arguments to run the test. Users can then write their own scripts
     # that call this script with the correct arguments and they do not need
     # to edit the file. The arguments use the variables in this file as their
     # defaults allowing the user to edit the file if that is their preferred
     # workflow.
-    tvacdata_dir = "/Users/sbogat/Documents/01_Research/Roman/corgidrp_workspace/CGI_TVAC_Data"
-    outputdir = "/Users/sbogat/Documents/01_Research/Roman/corgidrp_workspace/results"
+    tvacdata_dir = "/Users/kevinludwick/Library/CloudStorage/Box-Box/CGI_TVAC_Data/Working_Folder/"
+    outputdir = thisfile_dir
 
     ap = argparse.ArgumentParser(description="run the l2a->l2a_noisemap end-to-end test")
     ap.add_argument("-tvac", "--tvacdata_dir", default=tvacdata_dir,
                     help="Path to CGI_TVAC_Data Folder [%(default)s]")
     ap.add_argument("-o", "--outputdir", default=outputdir,
                     help="directory to write results to [%(default)s]")
     args = ap.parse_args()
+    args_here = ['--tvacdata_dir', tvacdata_dir, '--outputdir', outputdir]#, '--e2e_flag',False]
```

Comment:
are these just for debugging? should they be removed?

Reply:
Yeah, I used that for debugging so that I could do breakpoints (instead of running from the command line).

Reply:
But it only works if one uncomments `args = ap.parse_args(args_here)`.